lib/oref0-setup/mdt-cgm.json

```diff
@@ -125,15 +125,15 @@
   },
   {
      "get-bg": {
-      "command": "! bash -c \" (echo -n MDT cgm data retrieve && openaps monitor-cgm 2>/dev/null >/dev/null && grep -q glucose cgm/cgm-glucose.json && echo d) && cp -pu cgm/cgm-glucose.json cgm/glucose.json && cp -pu cgm/glucose.json monitor/glucose-unzoned.json && (echo -n MDT cgm data reformat && openaps report invoke monitor/glucose.json 2>/dev/null >/dev/null && echo ted)\""
+      "command": "! bash -c \" (echo -n MDT cgm data retrieve && openaps monitor-cgm 2>/dev/null >/dev/null && grep -q glucose cgm/cgm-glucose.json && echo d) && cp -pu cgm/cgm-glucose.json cgm/glucose.json && cp -pu cgm/glucose.json monitor/glucose-unzoned.json && (echo -n MDT cgm data reformat && openaps report invoke monitor/glucose.json nightscout/glucose.json 2>/dev/null >/dev/null && echo ted)\""
```

Comment:
Since we are using the nightscout/glucose.json instead of the monitor/glucose.json, this keeps those two files with the same CGM data. Otherwise nightscout/glucose.json is updaed by the ns-loop. So this keeps the two timestamps about the same with the same data.